### ISSUE TITLE
1.16.5 ghost blocks don't render 

### ISSUE THREAD

**Issue by Lupus590** [Direwolf20-MC/BuildingGadgets#609]
The building gadgets ghost blocks are not visible when playing on 1.16.5

* Forge version: 36.0.13
* Building Gadgets version:  3.7.3
* How to reproduce the issue: Use building gadget in build to me mode with stone bricks.

**Comment by Lupus590** [comment]
Placing does render correctly, it's just the mouse over visuals.

**Comment by MichaelHillcox** [comment]
What?

**Comment by Lupus590** [comment]
There's a highlight thing that shows where blocks will be placed when you right-click. This didn't seem to be working when I was running.

See this video of Dire, there's a projection of what the building gadget will do before he tells it to place. I'm not seeing that when I'm using the gadget. https://youtu.be/MrJ1jchxK3Q?t=486

**Comment by MichaelHillcox** [comment]
Are you using optifine? Any other rending mods? Anything that might change this fact. You can also try and use the mod on it's own and see if it's working without other mods.

**Comment by Lupus590** [comment]
It's still a problem with just Building Gadgets installed.

**Comment by MichaelHillcox** [comment]
I'll look into it

**Comment by Lupus590** [comment]
Just turned off the fabulous setting and it started working.

**Comment by MichaelHillcox** [comment]
Yeah, I can't really fix this, we don't yet have super good workarounds for Fabulous settings, I'll look into it but I don't think there is much I can do.
